lightspeed/flame
# Add slice and money-sent icons

## Description

We need to have access to two new icons on payment reports that are not available in Flame just yet.





## How to test?

- Checkout branch, run `yarn dev`
- Go to the [Storybook Icons](http://localhost:6006/?path=/story/components-icon--story)
- Check the `money-sent` and `slice` icons

## Checklist
